### PR TITLE
Add Kubewarden to integrations.yaml

### PR DESCRIPTION
**Pull request by flavio**
Kubewarden is using OpenTelemetry inside of its Rust and Go code both for tracing and metrics.

### CODE REVIEW

File: data/ecosystem/integrations.yaml [hunk 1]
```diff
@@ -238,3 +238,8 @@
   docsUrl: https://istio.io/latest/docs/tasks/observability/distributed-tracing/opentelemetry/
   components: [Go]
   oss: true
+- name: Kubewarden
```
Comment (theletterf) [inline data/ecosystem/integrations.yaml:241]: @flavio Could you add `Kubewarden` to the list of cSpell exceptions on line 1? Thanks!

Comment (flavio) [reply data/ecosystem/integrations.yaml:241]: done, change force pushed

**Comment by flavio** [comment]
There's still a check failing. I've run `npm run test-and-fix` as suggested [here](https://github.com/open-telemetry/opentelemetry.io/actions/runs/9515656884/job/26230480884?pr=4676), but no file was changed locally.

Can someone help with that?

**Review by cartermp — APPROVED** [review]

**Pull request merged**